src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/JSON2SolutionGraph.java

```diff
@@ -0,0 +1,18 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper;
+
+import org.apache.jena.atlas.json.io.parserjavacc.javacc.ParseException;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.atlas.json.JsonException;
+
+import se.liu.ida.hefquin.engine.federation.access.JSONResponse;
+
+public interface JSON2SolutionGraph {
```

Comment:
Add an action suffix to this name, so that someone that doesn't know the approach can know what this class does (e.g., `JSON2SolutionGraphConverter`